verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -47,8 +43,97 @@ const styles = () =>
         },
     });
 
+function initialValuesFromCase(c?: Case): NewCaseFormValues {
+    if (!c) {
+        return {
+            sex: undefined,
+            minAge: undefined,
+            maxAge: undefined,
+            age: undefined,
+            ethnicity: undefined,
+            nationalities: [],
+            profession: undefined,
+            location: undefined,
+            confirmedDate: null,
+            methodOfConfirmation: undefined,
+            onsetSymptomsDate: null,
+            firstClinicalConsultationDate: null,
+            selfIsolationDate: null,
+            admittedToHospital: undefined,
+            hospitalAdmissionDate: null,
+            icuAdmissionDate: null,
+            outcomeDate: null,
+            outcome: undefined,
+            symptoms: [],
+            transmissionRoutes: [],
+            transmissionPlaces: [],
+            transmissionLinkedCaseIds: [],
+            travelHistory: [],
+            sourceUrl: '',
+            notes: '',
+        };
+    }
+    return {
+        sex: c.demographics?.sex,
+        minAge:
+            c.demographics?.ageRange?.start !== c.demographics?.ageRange?.end
+                ? c.demographics?.ageRange?.start
+                : undefined,
+        maxAge:
+            c.demographics?.ageRange?.start !== c.demographics?.ageRange?.end
+                ? c.demographics?.ageRange?.end
+                : undefined,
+        age:
+            c.demographics?.ageRange?.start === c.demographics?.ageRange?.end
+                ? c.demographics?.ageRange?.start
+                : undefined,
+        ethnicity: c.demographics?.ethnicity,
+        nationalities: c.demographics?.nationalities,
+        profession: c.demographics?.profession,
+        location: c.location,
+        confirmedDate:
+            c.events.find((event) => event.name === 'confirmed')?.dateRange
+                ?.start || null,
+        methodOfConfirmation: c.events.find(
+            (event) => event.name === 'confirmed',
+        )?.value,
+        onsetSymptomsDate:
+            c.events.find((event) => event.name === 'onsetSymptoms')?.dateRange
+                ?.start || null,
+        firstClinicalConsultationDate:
+            c.events.find((event) => event.name === 'firstClinicalConsultation')
+                ?.dateRange?.start || null,
+        selfIsolationDate:
+            c.events.find((event) => event.name === 'selfIsolation')?.dateRange
+                ?.start || null,
+        admittedToHospital: c.events.find(
+            (event) => event.name === 'hospitalAdmission',
+        )?.dateRange?.start
```

Comment:
This is the value param of the hospitalAdmission event, so this can be:
```ts
admittedToHospital: c.events.find(
            (event) => event.name === 'hospitalAdmission',
        )?.value
```

Reply:
cool, simpler.